docker/install_dependencies.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+#  Copyright (C) 2018-2024 LEIDOS.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+set -e
+
+# Install software-proprties-common to be able to setup PPA repos
+sudo apt-get update
+sudo apt-get install -y software-properties-common
+
+sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
+sudo add-apt-repository -y ppa:deadsnakes/ppa
+sudo apt-get update
+
+# Download apt dependencies
+sudo apt-get install -y --allow-unauthenticated gcc-7 g++-7 python3.6 unzip tar python3.6-dev \
+  pkg-config sqlite3 autoconf libtool curl make libxml2 libsqlite3-dev \
+  libxml2-dev cmake libxerces-c-dev libfox-1.6-dev libgdal-dev libproj-dev \
+  libgl2ps-dev python3.7 python3-pip automake openjdk-11-jdk ant python3.7-dev \
+  python3.7-distutils x11-xserver-utils dconf-editor dbus-x11 libglvnd0 libgl1 \
+  libglx0 libegl1 libxext6 libx11-6 python3-dev \
+  build-essential pkg-config lbzip2 libprotobuf-dev protobuf-compiler patch rsync \
+  wget vim nano xterm libprotobuf-dev git
+sudo rm -rf /var/lib/apt/lists/*
+
+#update-alternatives --set python /usr/bin/python3.7
```

Comment:
we should remove it if we are not using it

Reply:
Yes

Reply:
Addressed